docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -3016,10 +3018,12 @@ public final class Target_XmlData {
 
 Some extensions may be private, and some may wish to be part of the
 broader Quarkus ecosystem, and available for community re-use.
+ifndef::no-quarkiverse[]
```

Comment:
again, why is existence of quarkiverse considered not relevant to know as a red hat build of quarkus?

Reply:
Sorry, my mistake. I've removed all changes to this file.